pages/urls/views/__init__.py

```diff
@@ -22,9 +22,10 @@
 # OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from ...compat import include, re_path
+from ...compat import include, re_path, path
 
 urlpatterns = [
+    path(r'app/sequence/', include('pages.urls.views.progress')),
```

Comment:
`app/sequence/` instead of `app/sequences/`? Have to check for consistency with other views, like `^editables/`.